### DISCUSSION TITLE
Taller Construyendo Aplicaciones Web Modernas y Escalables con Java

### DISCUSSION THREAD

**Discussion by ecabrerar** [JavaDominicano/java-dominicano-comite-ejecutivo#25] [General]
Agregar ideas sobre la organización del taller Construyendo Aplicaciones Web Modernas y Escalables con Java

**Comment by ecabrerar** [comment]
Sería mejor si lo ponemos la opción de crear una cuenta nueva, ellos dan un crédito de 200 dólares aquí https://cloud.digitalocean.com/registrations/new?refcode=f6fcd01aaffb 
@fredpena

**Comment by ecabrerar** [comment]
Queda investigar si te obligan a poner una tarjeta de crédito, si eses el caso, yo le puedo escribir en nombre de la comunidad y solicitarle créditos para los participantes del taller.

**Comment by ecabrerar** [comment]
@fredpena   Agregar la biografía tuya y de hector al README del taller.

**Comment by fredpena** [comment]
@hectorvent Para que este en la discusión.
